packages/fxa-auth-server/scripts/prune-tokens.ts

```diff
@@ -45,6 +45,11 @@ export async function init() {
       'Max age of token. Any tokens older than this value will be pruned, except session tokens with associated devices. A value of 0 results in a no-op.',
       '0'
     )
+    .option(
+      '--maxTokenAgeWindowSize <number>',
+      'The number of tokens to consider when pruning. This applies specifically to session tokens and has a limiting effect on the query. Increasing value will pull in more tokens for deletion.',
+      100000
```

Comment:
This should be 10k right?

Reply:
It turns out 10k wasn't nearly big enough for prod. At least that's what became apparent when we audited the production DB, so I think 100k is a good starting point, and using this as the default value _may_ result in no extra config changes on SRE's side. I don't foresee this large value having a negative impact on stage either.